target/linux/ath79/base-files/etc/board.d/02_network

```diff
@@ -230,6 +230,7 @@ ath79_setup_interfaces()
 		;;
 	tplink,archer-a7-v5|\
 	tplink,archer-c6-v2|\
+	tplink,archer-a6-v2|\
```

Comment:
Apply correct sorting, please.

Reply:
back to c6-v2-us, so it is fine to keep in same position?